deploy/ocm-agent-operator-managednotifications/10-managednotifications-upgrade-cr.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: ocmagent.managed.openshift.io/v1alpha1
+kind: ManagedNotification
+metadata:
+  name: sre-upgrade-managed-notifications
+  namespace: openshift-ocm-agent-operator
+spec:
+  notifications:
+    - name: UpgradeControlPlaneUpgradeTimeout
+      severity: Error
+      summary: "Control plane upgrade timeout"
+      activeBody: |-
+        SRE has observed that controlplane cannot be finished in the given time period, SRE will take action on the resolution. 
+      resendWait: 12
+      resolvedBody: |-
```

Comment:
I think actually we don't want to send a 'resolved' message for any of these. The cluster owner will already get this message via other means when the cluster finishes upgrading, this would just be adding additional SL spam.

The catch is that currently OCM Agent will always send a resolved SL right now, even if "ResolvedBody" is empty. This should change. I've created OSD-13528 to track that work.

I think what we will need to perhaps do is enable ocm-agent to handle the ability to optionally send a 'resolved' message. (ie. if the resolvedBody is empty, then do not send a SL)

Reply:
@mrbarge I currently leave it here since we still need this right? So after OSD-13528 is resolved we can get rid of this part. BTW, I have resolved all other comments and please help to check. I also did the test. I will leave another comment for testing results.